ocs/ocp.py

```diff
@@ -53,7 +53,7 @@ def exec_oc_cmd(self, command):
                 without the initial 'oc' at the beginning
 
         Returns:
-            Munch Obj: this object represents a returned yaml file
+            Munch Obj if the object represents a returned yaml file, else returns a list
```

Comment:
When it will return a list?

Reply:
The output from Ceph commands is either in the form of dictionary or list. I've seen this in a couple of occasions.

The following is one such case.
```
sh-4.2# rbd ls rbd --format json 
["pvc-29e8c2cf-85d4-11e9-a278-0a334edd4bdc","pvc-771654f7-85d3-11e9-9454-02a058fe7dec"]
```
converting such an output to dict fails.
```
>   return [item.toDict() for item in out if item]
E   AttributeError: 'str' object has no attribute 'toDict'
```

Reply:
Would it be better to always return a list than to have two different return types? Even if the list has one item, I think the consistency would be better than having any code that consumes this need to consider both cases.

Reply:
I probably missing something.
This method is executing oc command so how 
```python
sh-4.2# rbd ls rbd --format json
```
is related to it?